src/templates/pages/server_status.html

```diff
@@ -2,8 +2,70 @@
 {% load staticfiles %}
 
 {% block extra_head %}
-    <script src="https://bernii.github.io/gauge.js/dist/gauge.min.js"></script>
 {% endblock %}
+
 {% block content %}
-    <server_status></server_status>
+    <div class="ui container">
+        <h1>Recent submissions (up to 250 or 2 days old)</h1>
+        <table class="ui table">
+            <thead>
+            <th>Competition</th>
+            <th>Submitter</th>
+            <th>Submitted at</th>
+            <th>Status</th>
+            </thead>
+            <tbody>
+                {% for sub in submissions %}
+                <tr>
+                    <td>{{ sub.phase.competition }}</td>
```

Comment:
maybe also add submission pk to this list? so we could go find the sub in admin/shell plus or something and get task id and send kill signal if we need to?

Reply:
✔️